match/lib/match/runner.rb

```diff
@@ -166,7 +166,7 @@ def fetch_provisioning_profile(params: nil, certificate_id: nil, app_identifier:
 
       if params[:force_for_new_devices] && !params[:readonly]
         if prov_type != :appstore
-          params[:force] = device_count_different?(profile: profile, keychain_path: keychain_path, platform: params[:platform].to_s) unless params[:force]
+          params[:force] = device_count_different?(profile: profile, keychain_path: keychain_path, platform: params[:platform].to_sym) unless params[:force]
```

Comment:
Ugh, interesting one! So the param is a string and has to be converted to a symbol?

Reply:
Seems like it—wasn't able to get the switch to work without the `to_sym` call (kept hitting the else clause)